### ISSUE TITLE
[Bug] Fix slack screen sharing

### ISSUE THREAD

**Issue by deepanchal** [deepanchal/nixos#9]

**Comment by deepanchal** [comment]
References:
- https://github.com/outfoxxed/hy3/issues/69
- https://github.com/hyprwm/xdg-desktop-portal-hyprland/issues/251
- https://github.com/hyprwm/xdg-desktop-portal-hyprland/issues/248
- https://github.com/hyprwm/xdg-desktop-portal-hyprland/issues/247
- https://github.com/hyprwm/xdg-desktop-portal-hyprland/issues/178

**Comment by piyushyadav0191** [comment]
Is it a big issue now?

**Comment by deepanchal** [comment]
It's still not working for me. I am just collecting references @piyushyadav0191 
Screensharing on swaywm works great with xdg-desktop-portal-wlr but doesn't work on hyprland :(

**Comment by piyushyadav0191** [comment]
@deepanchal I have tried numerous solution with xdph, xdp-wlr, aquamarine but no luck. Let me know if there is any solution for this prob

**Comment by spikespaz** [comment]
I see multiple issues that look like this, but none of them have described the following:

Sharing the screen works if the program that starts the portal is on the screen which I want to share.

Sharing the screen does not work if I share a screen which does not contain the current active window.

Can anybody reproduce?

**Comment by jficz** [comment]
> Can anybody reproduce?

No, it's quite random for me: https://github.com/hyprwm/xdg-desktop-portal-hyprland/issues/11#issuecomment-2413484426
